test/terra/backends/aer_simulator/test_truncate.py

```diff
@@ -135,3 +137,32 @@ def test_delay_measure(self):
         metadata = result.results[0].metadata
         self.assertIn('measure_sampling', metadata)
         self.assertTrue(metadata['measure_sampling'])
+
+    def test_parameterized_circuits(self):
+        """Test truncation for parameterized circuits"""
+        backend = self.backend()
+        circuit = QuantumCircuit(10, 10)
+        circuit = circuit.compose(RealAmplitudes(5), range(5))
+        circuit = transpile(circuit, backend)
```

Comment:
If there's non-trivial transpilation being done here, it wouldn't hurt to seed the transpiler as well, but in practice it's _probably_ ok - I think all the random elements are moot for Aer because it's got all-to-all connectivity.

Reply:
I changed this [test circuit](https://github.com/Qiskit/qiskit-aer/blob/47f2765b8c117e8f29792c45f50f64b80b32136f/test/terra/backends/test_parameterized_qobj.py#L303-L312) for lightweight execution.